src/main/java/core/basesyntax/impl/WritingToNewFileImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.WritingToNewFile;
+import core.basesyntax.db.FruitShopDao;
+import core.basesyntax.service.CantWorkWithThisFileException;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WritingToNewFileImpl implements WritingToNewFile {
```

Comment:
The class name `WritingToNewFileImpl` does not clearly reflect its responsibility. Consider renaming it to something more descriptive, like `FruitStorageWriter` or `FruitReportWriter`, to be more in line with its role of writing fruit storage data to a file.